### DISCUSSION TITLE
Import without Logo

### DISCUSSION THREAD

**Discussion by amadron** [ScanMountGoat/ldr_tools_blender#23] [General]
Hi, 

pretty good addon.  Tested it with bricklink Studio and works like a dream. 

I wanted to ask if there is a way to import the models without the Lego Logo? Because it adds a lot of geometry to the models.

Best regards

amadron

**Comment by ScanMountGoat** [comment]
I haven't added selecting the stud type to the addon settings yet. This is being tracked by https://github.com/ScanMountGoat/ldr_tools_blender/issues/3.

**Comment by amadron** [comment]
Ah ok, thank you for your reply :).
